docs/presentation/Content/PropertySpecification.md

```diff
@@ -1,37 +1,138 @@
+| --------------------------------------------------------------------------------------------------- | --------- | ----------------------------------------------------------------- | ----------- |
+| [`name`](#attribute-name)                                                                           | Yes       | `string`                                                          |             |
+| [`overridesPriority`](#attribute-overridespriority)                                                 | No        | `number`                                                          | `1000`      |
+| [`labelOverride`](#attribute-labeloverride)                                                         | No        | `string`                                                          | `undefined` |
+| [`categoryId`](#attribute-categoryid)                                                               | No        | `string \| CategoryIdentifier`                                    | `undefined` |
+| [`isDisplayed`](#attribute-isdisplayed)                                                             | No        | `boolean`                                                         | `undefined` |
+| [`doNotHideOtherPropertiesOnDisplayOverride`](#attribute-donothideotherpropertiesondisplayoverride) | No        | `boolean`                                                         | `false`     |
+| [`renderer`](#attribute-renderer)                                                                   | No        | [`RendererSpecification`](./RendererSpecification.md)             | `undefined` |
+| [`editor`](#attribute-editor)                                                                       | No        | [`PropertyEditorSpecification`](./PropertyEditorSpecification.md) | `undefined` |
+
+### Attribute: `name`
+
+Name of the ECProperty to apply overrides to. A `"*"` may be specified to match all properties in current context. The current context is determined based on where the override is specified:
+
+- When used in a [content modifier](./ContentModifier.md#attribute-propertyoverrides), the properties of the ECClass specified by the [`class` attribute](./ContentModifier.md#attribute-class) are used.
+- When used in one of the [content specifications](./ContentRule.md#attribute-specifications), properties produced by that specification are used.
+
+### Attribute: `overridesPriority`
+
+> **Default value:** `1000`
+
+There may be multiple property specifications that apply to a single property and there may be conflicts between different attributes. The `overridesPriority` attribute is here to help
+solve the problem - if multiple specifications attempt to override the same attribute, the override of specification with highest `overridesPriority` value is used. The order of overrides
+from specification with the same `overridesPriority` is undefined.
```

Comment:
```suggestion
from specification with the same `overridesPriority` is defined by the order they appear in the overrides list
```

Recent changes should make it so the overrides are added in a priority queue style. If two overrides are of the same priority, they should appear in the order they were added.